Server/docker-compose.yaml

```diff
@@ -67,7 +67,8 @@ services:
                 --seeds tcp://node-002.creditcoin.org:8800 \
                 --seeds tcp://node-003.creditcoin.org:8800 \
                 --maximum-peer-connectivity 5 \
-                --scheduler parallel'
+                --scheduler serial \
+                --state-pruning-block-depth 2000000'
```

Comment:
I think this has to be merged after the 2.0 release, since `state-pruning-block-depth` isn't in Sawtooth 1.0.5 and will likely break existing nodes if added on a 1.0.5-based node